README.md

```diff
@@ -17,8 +17,123 @@ value from data whilst ensuring it is properly governed.
+````
+
+# Postgres Connection Details
+After creating the Egeria Integration Daemon server that will host the postgres connector, the next step is to configure the connector. 
+As typical with Egeria the we configure the postgres connector by posting a JSON configuration document to a REST API endpoint. 
+
+In this readme we'll break down a cURL request, which is based on the following command from the Postman collection.
+`Configure integration daemon server with an integration service`
+
+Below we can see the typical REST endpoint and the associated header where 
+`platformName` is the name of the server hosting the egeria platform which hosts the Integration Daemon, and
+`integrationDaemonServer` is the name of the server that is to host the postgres connector.
+````shell
+ curl --location --request POST 'https://platformName:9443/open-metadata/admin-services/users/garygeeke/servers/integrationDaemonServerName/integration-services/database-integrator' \
+--header 'Content-Type: application/json' \
+````
+
+In the above section the
+omagserverPlatformRootURL is the name of the OMAG platform that is hosting the OMAG Server that hosts the OMAS that the integration daemon is partnered with.
+omagserverName is the name of OMAG server that is hosting the OMAS that this integration daemon is partnered with. These fields contain the information necessary
+to connect to the Egeria Server which is hosting the Data Manager OMAS.
+
+```shell
+
+--data-raw '{
+    "class": "IntegrationServiceRequestBody",
+    "omagserverPlatformRootURL": "https://platformName:9443",
+    "omagserverName" : "omagServer",
+    
+  ````
+In the next section we see how to configure the name of this particular connector instance and the userID that will be used to connect to the 
+egeria OMAS server.
+
+```shell
+    "integrationConnectorConfigs" : [ 
+        {
+             "class": "IntegrationConnectorConfig",
+             "connectorName" : "connectorInstanceName",             
+             "connectorUserId" : "egeriaUserID",
+  ````   
+In the next section we declare a connection object that will provide the details necessary to make a connection to the postgres database.
+We also provide the fully qualified Java classname for the postgres connectors provider.
+The `recognizedConfigurationProperties` and the `configurationProperties`sections provides the names of any parameters we wish to pass to the postgres JDBC client.
+In this example we just pass the jdbc connection url and set ssl to false. The full list of possible parameters are detailed in the postgres documentation 
+[here.](https://jdbc.postgresql.org/documentation/head/connect.html)
+```shell
+  "connection" : 
+             { 
+                 "class" : "Connection",
+                 "userId" : "postgresUserID",
```

Comment:
what permissions does this user need

Reply:
closing this for now - when we find/understand more needs to be doced can be done through another pr